tests/test_wrappers.py

```diff
@@ -14,3 +13,10 @@ def test_content_md5_check_raises_exception_if_fails():
+    parsed_json = DictWrapper(xml).parsed
+    assert parsed_json == expected_json
```

Comment:
Just to be clear, these are not JSON responses: these are dict-like objects. In fact, `.parsed` is dict-like, but not strictly a dict object, so the equivalence check here might be iffy.